jobs/keepalived/spec

```diff
@@ -20,8 +20,8 @@ properties:
     description: when health check fails, this triggers a fail over. The default command checks the haproxy process is still alive.
     default: killall -0 haproxy
   keepalived.interface:
-    description: interface keepalived will use to mount the VIP
-    default: eth0
+    description: interface keepalived will use to mount the VIP. If set to 'auto', uses the default interface on the VM
```

Comment:
What if we check the vip, instead? This way the box would find the most appropriate interface to use for the IP it needs to add.